lib/src/sidenav/Sidenav.stories.tsx

```diff
@@ -41,11 +54,17 @@ export const Chromatic = () => (
           </DxcSidenav.Group>
         </DxcSidenav.Section>
         <DxcSidenav.Section>
-          <DxcSidenav.Group collapsable={true} title="Section Group" icon={iconSVG}>
+          <DxcSidenav.Group
+            collapsable={true}
+            title="Section Group"
+            icon="https://cdn-icons-png.flaticon.com/512/5039/5039041.png"
+          >
             <DxcSidenav.Link selected>Group Link</DxcSidenav.Link>
             <DxcSidenav.Link icon={iconSVG}>Group Link</DxcSidenav.Link>
           </DxcSidenav.Group>
-          <DxcSidenav.Link icon={iconSVG}>Single Link</DxcSidenav.Link>
+          <DxcSidenav.Link icon="https://upload.wikimedia.org/wikipedia/commons/7/73/Flat_tick_icon.svg">
```

Comment:
The newWindow prop should be added to one of the links to have a test showing the external link.

Reply:
Done!